### PR TITLE
Grid - basic functionality

### LINKED ISSUE #1: Make renderer for current data state
The data of a nonogram will be represented for rendering as a color array, and a two-dimensional array of integers,
where -1 will be a cell surely empty, and 0 currently unknown, others being colors

### PR DESCRIPTION
**Pull request by Frendzlu**
Clickable 10x10 grid, waiting for color inputs and size inputs, size currently temporarily hardcoded

closes #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,11 +3,23 @@
...
+    <link rel="stylesheet" href="style.css">
+    
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + Svelte + TS</title>
   </head>
+
+  
+
+
+
```
Comment (qualv13) [inline index.html:15]: delete unnecessary blank lines in this file

File: src/lib/Tile.svelte [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        let ctx = (e as HTMLCanvasElement).getContext("2d")!
+        ctx.moveTo(0, 0)
+        ctx.lineTo(size, size)
+        ctx.moveTo(0, size)
+        ctx.lineTo(size, 0)
+    }
+</script>
+
+<div class="cell" aria-pressed="false" role="button" tabindex="0"
+    on:click={clickHandle}  on:keydown={() => {}}
```
Comment (qualv13) [inline src/lib/Tile.svelte:20]: put all attributes in seperate lines

File: src/lib/Tile.svelte [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    style={`background-color: ${translateColor(color >= 0 ? color : 0)}; width: ${size}px; height: ${size}px`}>
+    <canvas width={size} height={size} class:active={isCrossed} on:load={(e) => drawCross(e.currentTarget)}></canvas>
```
Comment (qualv13) [inline src/lib/Tile.svelte:22]: canvas is unnecessary here, use svg

File: style.css
Comment (qualv13) [inline style.css:1]: why

**Review by qualv13 — CHANGES_REQUESTED** [review]

File: src/lib/Tile.svelte (hunk 3 above)
Comment (Frendzlu) [reply src/lib/Tile.svelte:28]: to be done at later date, see issue #9

File: style.css
Comment (Frendzlu) [reply style.css:1]: badly done merge, fixed

**Review by qualv13 — APPROVED** [review]

**Pull request merged**